packages/signer/signer-btc/src/internal/app-binder/command/GetWalletAddressCommand.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  type Apdu,
+  ApduBuilder,
+  ApduParser,
+  type ApduResponse,
+  type Command,
+  type CommandResult,
+  CommandResultFactory,
+  InvalidStatusWordError,
+  isCommandErrorCode,
+} from "@ledgerhq/device-management-kit";
+
+import {
+  type GetWalletAddressCommandArgs,
+  type GetWalletAddressCommandResponse,
+} from "@api/app-binder/GetWalletAddressCommandTypes";
+
+import {
+  BitcoinAppCommandError,
+  bitcoinAppErrors,
+} from "./utils/bitcoinAppErrors";
+
+export class GetWalletAddressCommand
+  implements
+    Command<GetWalletAddressCommandResponse, GetWalletAddressCommandArgs>
+{
+  args: GetWalletAddressCommandArgs;
+
+  constructor(args: GetWalletAddressCommandArgs) {
+    this.args = args;
+  }
```

Comment:
```suggestion
  constructor(private readonly args: GetWalletAddressCommandArgs) {}
```
This will have the same effect and more clear, you can access to the property by `this.args` just as in the other places (most) in the code base.